networks/59141/markets.json

```diff
@@ -0,0 +1,7 @@
+[
+  "spot://ETH/USDC",
+  "spot://WETH/USDC",
```

Comment:
we don't need it, as we have etc

Reply:
badbcb5cbf000eadc1f7e12992055a838db1ef84